apps/web/src/env.mjs

```diff
@@ -76,6 +77,7 @@ export const env = createEnv({
     DATABASE_URL: process.env.DATABASE_URL,
     DIRECT_URL: process.env.DIRECT_URL,
     FEEDBACK_WEBHOOK_URL: process.env.FEEDBACK_WEBHOOK_URL,
+    BLOB_DATA_API_KEY: process.env.BLOB_DATA_API_KEY,
```

Comment:
I think an extra variable BLOB_DATA_API_ENABLED would be useful for our case. We are incurring in extra charges because of this endpoint, so we want to disable it, not even keep it behind authentication. However, if another team wants to spin up a blobscan instance, they can decide if enable/disable and authenticated/public.

Reply:
added